src/Rules/UnusedViewsRule.php

```diff
@@ -5,6 +5,7 @@
 namespace Larastan\Larastan\Rules;
 
 use Illuminate\View\Factory;
+use Illuminate\View\ViewFinderInterface;
```

Comment:
I don't see where this interface is used 🤔

Reply:
my mistake, a holdover from an original approach before i learned that `->getFinder()` existed